### ISSUE TITLE
Graphs / Charts

### ISSUE THREAD

**Issue by HamburgerJungeJr** [jobsta/reportbro-designer#6]
Hi,

are you planning to implement basic charts, like Pie-, Graph or Bar-chart.

As far as I have seen, there is no simple solution, like adding Chart.js for example, because there is no Python equivalent.

**Comment by alhman** [comment]
Hi,

at the moment we have no plans to implement charts. But in the long term this would definitely be a nice addition. You're right that we'd need a chart library for the designer (javascript) and the lib (python). If we use different libs the preview in the designer layout will look different (don't know if this would be an issue). e.g. chart.js for the designer and matplotlib for the lib.

**Comment by cristi-boboc** [comment]
Hi,

I believe you can achieve this by having the backend generating the chart as an image, and then loading the image from url.

Hope it helps

**Comment by alhman** [comment]
you're right. Adding the chart generation to the backend should be sufficient.The designer can request a chart update for preview on demand.
